src/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+import { configureStore } from "@reduxjs/toolkit";
+import { Provider } from "react-redux";
+import todo from "../src/features/todo"
+import newTask from './features/newTask';
+
+const store = configureStore({
+  reducer: {
+    todos: todo,
+    newTask: newTask
+  }
+})
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <Provider store={store}>
+    <App />
+    </Provider>
+  </React.StrictMode>
```

Comment:
This `StrictMode` could be extracted and imported from react, instead of using the dot notation to access it.